examples/gpt-2/utils/processor.py

```diff
@@ -11,6 +11,7 @@
 import regex as re
 from functools import lru_cache
 
+
 @lru_cache()
 def bytes_to_unicode():
```

Comment:
Have the functions in this file been included in the GPT2 tokenizer etc as in Texar-PyTorch? So that we can get rid of this file?

Reply:
Yes. I have already replaced this part by `GPT2Tokenizer` in https://github.com/asyml/texar/pull/228 

After this one is merged, I will refactor and fix lint error in that PR

Reply:
Yes. After this one is merged, I will refactor https://github.com/asyml/texar/pull/228 and merge it at first and then polish GPT2 example.